frontend/packages/console-shared/src/promql/cluster-dashboard.ts

```diff
@@ -44,9 +44,9 @@ const top25Queries = {
   [OverviewQuery.NODES_BY_MEMORY]:
     'topk(25, sort_desc(node_memory_MemTotal_bytes - node_memory_MemAvailable_bytes))',
   [OverviewQuery.NODES_BY_STORAGE]:
-    'topk(25, sort_desc(avg_over_time(sum(node_filesystem_size_bytes{instance="<%= node %>",fstype!="tmpfs",mountpoint!="/boot|/boot/efi"}[5m] - node_filesystem_avail_bytes{instance="<%= node %>",fstype!="tmpfs",mountpoint!="/boot|/boot/efi"}[5m]))))',
+    'topk(25, sort_desc(sum(node_filesystem_size_bytes{instance=~".*",fstype!="tmpfs",mountpoint!="/boot|/boot/efi"} - node_filesystem_avail_bytes{instance=~".*",fstype!="tmpfs",mountpoint!="/boot|/boot/efi"}) BY (instance)))',
```

Comment:
`instance=~".*"` is the same as having no selector at all on `instance` so it can be removed.

```suggestion
    'topk(25, sort_desc(sum(node_filesystem_size_bytes{fstype!="tmpfs",mountpoint!="/boot|/boot/efi"} - node_filesystem_avail_bytes{fstype!="tmpfs",mountpoint!="/boot|/boot/efi"}) BY (instance)))',
```